File: ce/snippets/csharp/CRMV8/metadataquery/cs/entityproperties.cs
// <snippetentityproperties>


 //A properties expression to limit the properties to be included with entities
 MetadataPropertiesExpression EntityProperties = new MetadataPropertiesExpression()
 {
  AllProperties = false
 };
 EntityProperties.PropertyNames.AddRange(new string[] { "Attributes" });

// </snippetentityproperties>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/metadataqueryvb/vb/entityproperties.vb
' <snippetentityproperties>


 'A properties expression to limit the properties to be included with entities
 Dim EntityProperties As New MetadataPropertiesExpression() With {.AllProperties = False}
 EntityProperties.PropertyNames.AddRange(New String() { "Attributes" })

' </snippetentityproperties>